### ISSUE TITLE
Links stop working with using Mantine Datatable

### ISSUE THREAD

**Issue by swapnil233** [icflorescu/mantine-datatable#429]
**Describe the bug**

Other links do not work when mantine datatable is present. I am working in a Next.js 12 (pages) app. Whenever I put a `<DataTable>` component in a page, none of the other links on the page work (and also any links in any cells of the table). It hangs the entire app, and I have to close the tab and restart the server for things to work again, but as soon as I get to the page with the DataTable component, everything stops working again.

No idea what the issue is and it looks like no one else has run into it...

All the links within the app use Next.js's `<Link>` component, not a regular `a` tag.

Here's the data table component:

```typescript
import { NoteWithTagsAndCreator } from "@/types";
import { Avatar, Badge, Group, Stack, Text } from "@mantine/core";
import sortBy from "lodash/sortBy";
import { DataTable, DataTableSortStatus } from "mantine-datatable";
import Link from "next/link";
import { useRouter } from "next/router";
import { useEffect, useState } from "react";

interface INotesDataTable {
  notes: NoteWithTagsAndCreator[];
}

export default function NotesDataTable({ notes }: INotesDataTable) {
  const router = useRouter();
  const { teamId, projectId } = router.query;

  const [records, setRecords] = useState(sortBy(notes, "createdAt"));

  const [sortStatus, setSortStatus] = useState<DataTableSortStatus>({
    columnAccessor: "createdAt",
    direction: "asc",
  });

  useEffect(() => {
    const data = sortBy(
      notes,
      sortStatus.columnAccessor
    ) as NoteWithTagsAndCreator[];
    setRecords(sortStatus.direction === "desc" ? data.reverse() : data);
  }, [sortStatus, notes]);

  return (
    <DataTable
      withBorder
      withColumnBorders
      records={records}
      columns={[
        {
          accessor: "id",
          title: "ID",
          width: 80,
          render: (note) => notes.indexOf(note) + 1,
        },
        {
          accessor: "text",
          title: "Note",
          width: "40%",
          render: (note) => note.text.trim(),
        },
        {
          accessor: "tags",
          title: "Tags",
          render: (note) =>
            note.tags.map((tag) => (
              <Link
                key={tag.id}
                href={...}
                style={{
                  textDecoration: "none",
                }}
              >
                ...
              </Link>
            )),
        },
        {
          accessor: "createdBy",
          title: "Created by",
          sortable: true,
          render: (note) => (
            <Group position="apart" w={"100%"}>
              <Link
                href={...}
                style={{
                  textDecoration: "none",
                }}
              >
                ...
          ),
        },
        {
          accessor: "createdAt",
          title: "Date created",
          sortable: true,
          render: (note) => new Date(note.createdAt).toLocaleString(),
        },
      ]}
      sortStatus={sortStatus}
      onSortStatusChange={setSortStatus}
    />
  );
}

```

**Comment by icflorescu** [comment]
Hey @swapnil233, I'm afraid I could not reproduce the issue.
Have you, by any chance, upgraded to V7 yet? If so, do you still encounter the issue?

**Comment by icflorescu** [comment]
Will close this for now, as no one else encountered this.
Please feel free to reopen if you can come up with a working reproduction (codesandbox or something similar) and/or if you're still facing the issue with V7.

**Comment by dave-remedi8** [comment]
I am having the same issue? Any ideas?

**Comment by dave-remedi8** [comment]
I suspect it is my styles setup? Here is reproducible example. 

https://github.com/dave-remedi8/mantine-datatable-next-link

Thank you!

**Comment by dave-remedi8** [comment]
dropping down to `"@mantine/core": "7.7.0"` from `"@mantine/core": "7.7.1"` fixes this.

**Comment by icflorescu** [comment]
@dave-remedi8 There was a temporary bug in the very specific Mantine version **7.7.1** that generated quite a few issues [such as this one](https://github.com/icflorescu/mantine-datatable/issues/564) in a lot of projects; tt was fixed in Mantine 7.7.2.
